docs/user/alerts.md

```diff
@@ -60,6 +61,28 @@ the Machine API resources to discover the root cause. For more information on
 why nodes, or machines, might not become ready please see the
 [Machine API FAQ](https://github.com/openshift/machine-api-operator/blob/master/FAQ.md).
 
+## ClusterAutoscalerExcessiveUnregisteredNodeCleanup
+The cluster autoscaler has removed more than 1 node that failed to register within the
+last 6 hours. This indicates that that the autoscaler has been replacing nodes which
+are failing to join the cluster.
+
+### Query
+```
+rate(cluster_autoscaler_old_unregistered_nodes_removed_count[6h]) > 1
+```
+
+### Possible Causes
+* MachineSet, or cloud credential, configuration is causing machines to fail.
+* Network disruption is preventing machines from becoming nodes.
+
+### Resolution
+* Look at the cluster autoscaler events, or logs, to determine which Machines have been removed
+  and to which MachineSets they belong.
+* Ensure proper configuration for any MachineSets which are affected.
```

Comment:
We could prob add a helper command here?

Reply:
sure that make sense to me, what did you have in mind? (eg something for the events or logs or ?)